src/components/shared/Tray/TrayComponents/AutoComplete/AutoComplete.js

```diff
@@ -30,24 +31,29 @@ const AutoComplete = () => {
         );
 
       case 'train':
-        return (
+        // Hide train autoCompletes for future dates until api issues have been sorted
+        return whenState.when === 'now' ? (
           <>
             {autoCompleteTitle('Trains between')}
             <TrainAutoComplete />
             {autoCompleteTitle('and')}
             <TrainAutoComplete to />
           </>
+        ) : (
+          <div />
         );
```

Comment:
```suggestion
        return whenState.when === 'now' && (
          <>
            {autoCompleteTitle('Trains between')}
            <TrainAutoComplete />
            {autoCompleteTitle('and')}
            <TrainAutoComplete to />
          </>
        );
```

If it's a one sided if statement, better to do like the above rather than rendering a blank div.